augur/application/db/util.py

```diff
@@ -43,48 +53,47 @@ def execute_session_query(query, query_type="all"):
     return catch_operational_error(func)
 
 
-
-def convert_orm_list_to_dict_list(result):
-    new_list = []
+def convert_orm_list_to_dict_list(result: List[Any]) -> List[Dict[str, Any]]:
+    """
+    Converts a list of ORM model instances to a list of dictionaries.
+    """
+    new_list: List[Dict[str, Any]] = []
 
     for row in result:
-        row_dict = row.__dict__
-        try:
-            del row_dict['_sa_instance_state']
-        except:
-            pass
-        
+        row_dict = dict(row.__dict__)  # Copy to avoid mutating the ORM instance
+        row_dict.pop("_sa_instance_state", None)  # Remove SQLAlchemy internal state
         new_list.append(row_dict)
-    
-    return new_list
 
+    return new_list
 
 
-def convert_type_of_value(config_dict, logger=None):
-        
-    data_type = config_dict["type"]
+def convert_type_of_value(
+    config_dict: Dict[str, Any], logger: Optional[Any] = None
+) -> Dict[str, Any]:
+    """
+    Converts the 'value' field in config_dict to the type specified in 'type'.
+    Supported types: str, int, bool, float.
+    """
+    data_type: Optional[str] = config_dict.get("type")
 
-    if data_type == "str" or data_type is None:
+    if data_type is None or data_type == "str":
```

Comment:
why was the order of these changed? this could cause a crash if the data type is none because it is relying on the short curcuiting behavior of `or`